app/client/views/login/login.html

```diff
@@ -1,5 +1,7 @@
 <div id="particles-js"></div>
 <div id="login">
+  <script> var bLazy = new Blazy(); </script>
```

Comment:
Can this be moved to `loginCtrl.js`

Reply:
I think so. It didn't work for node modules but with bower I'm sure it will work